code-challenges.js

```diff
@@ -3,7 +3,15 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+const fibonacci = (n) =>{
+  let arr = [0,1];
+  for (let i = 2; i < 11; i++){
+    arr.push(arr[i - 2] + arr[i -1])
+  }
+ return arr.slice(1)
+}
 
+console.log(fibonacci())
```

Comment:
yay good work! just look at your first line: you are setting up your function to take in an argument of `n`, but you don't refer to `n` anywhere in your logic. you could either take that argument out, or find a way to use it to make your code able to return any number of values of the fibonacci sequence.